runtimes/peregrine/Cargo.toml

```diff
@@ -70,7 +70,7 @@ pallet-indices.workspace = true
 pallet-membership.workspace = true
 pallet-preimage.workspace = true
 pallet-proxy.workspace = true
-pallet-randomness-collective-flip.workspace = true
+pallet-insecure-randomness-collective-flip.workspace = true
```

Comment:
This pallet is not needed here. I think the renaming should also make it clear that we should remove it. :D

Reply:
Removed it with a migration: https://github.com/KILTprotocol/kilt-node/pull/500/commits/6187dc92d833bc35c144b9603eb8e46b28e83bca